packages/integration-stable/testRunner.ts

```diff
@@ -0,0 +1,40 @@
+import createClient from "../driver/src/index.node";
```

Comment:
BTW, some of these test setup files can probably move into some shared place like `driver/test/testUtil`, but I wanted to err on the side of copy-paste for now. Open to doing it now if this feels too WET.